### PR TITLE
Fix app reset after mobile restart

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2267

### CODE REVIEW

**Review by shamilkhan — DISMISSED** [review]

File: src/Components/Pages/TransferPage.tsx [hunk 1]
```diff
@@ -533,7 +533,7 @@ const TransferPage = () => {
       <div className={classes.wrapper}>
         <div className={classes.header}>
           {process.env.REACT_APP_MAINTENANCE === "false" &&
-            (!isReady ? (
+            (!isReady && !transactionStatus ? (
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: Could you assign the whole expression to a variable with a good name?

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 2]
```diff
@@ -162,9 +162,9 @@ export const EVMHomeAdaptorProvider = ({
     });
 
     wallet?.provider?.on("accountsChanged", (accounts: string[])=> {
-      console.log("Accounts changed:", { account, accounts });
-      if (walletSelected) setWalletType("unset");
-      setAccount(accounts[0])
+      console.log("Accounts changed:", { walletSelected, account, accounts });
+      if (walletSelected && account && account !== accounts[0]) setWalletType("unset");
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Could you assign the whole expression to a variable with a good name?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 ## vNext
 - Fix WalletConnect login issue after auth popup was closed
+- Fixed app transfer page reset after mobile restart
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md:6]: Use single pattern

Comment (DmitriyMolch) [reply CHANGELOG.md:6]: Before we used a pattern with `-ed` comment.  Do you want to switch to the pattern with or without `-ed` ending?

File: src/Components/Pages/TransferPage.tsx (hunk 1 above)
Comment (DmitriyMolch) [reply src/Components/Pages/TransferPage.tsx]: Fixed

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 2 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Fixed

File: CHANGELOG.md (hunk 3 above)
Comment (DmitriyMolch) [reply CHANGELOG.md:6]: Fixed

**Review by AndreiNavoichyk — DISMISSED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**